Aztec/Classes/Public/TextKit/TextList.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import UIKit
+
+
+// MARK: - Text List
+//
+class TextList
+{
+    // MARK: - Nested Types
+
+    /// Kind of Lists
+    ///
+    enum Kind {
+        case Ordered
+        case Unordered
+    }
+
+    // MARK: - Properties
+
+    /// Kind of List: Ordered / Unordered
+    ///
+    let kind: Kind
```

Comment:
I know this comment is going to sound very annoying, as it's extremely nitpicky and kind of subjective.  But `type` sounds like a more commonly-used term for this kind of differentiation.

Feel free to accept or ignore, though, as I know it means a tons of changes right now.